src/utils/loadScripts.py

```diff
@@ -1,180 +1,362 @@
+    logger.info("Commit: %s", settingsDict["SHORT_COMMIT_ID"])
+    logger.info("")
+    logger.info(
+        "%s | Removing failed downloads (%s)",
+        str(settingsDict["REMOVE_FAILED"]),
+        "REMOVE_FAILED",
+    )
+    logger.info(
+        "%s | Removing failed imports (%s)",
+        str(settingsDict["REMOVE_FAILED_IMPORTS"]),
+        "REMOVE_FAILED_IMPORTS",
+    )
+    if (
+        settingsDict["REMOVE_FAILED_IMPORTS"]
+        and not settingsDict["FAILED_IMPORT_MESSAGE_PATTERNS"]
+    ):
+        logger.verbose(
+            "> Any imports with a warning flag are considered failed, as no patterns specified (%s).",
+            "FAILED_IMPORT_MESSAGE_PATTERNS",
+        )
+    elif (
+        settingsDict["REMOVE_FAILED_IMPORTS"]
+        and settingsDict["FAILED_IMPORT_MESSAGE_PATTERNS"]
+    ):
+        logger.verbose(
+            "> Imports with a warning flag are considered failed if the status message contains any of the following patterns:"
+        )
+        for pattern in settingsDict["FAILED_IMPORT_MESSAGE_PATTERNS"]:
             logger.verbose('  - "%s"', pattern)
-    logger.info('%s | Removing downloads missing metadata (%s)', str(settingsDict['REMOVE_METADATA_MISSING']), 'REMOVE_METADATA_MISSING') 
-    logger.info('%s | Removing downloads missing files (%s)', str(settingsDict['REMOVE_MISSING_FILES']), 'REMOVE_MISSING_FILES')
-    logger.info('%s | Removing orphan downloads (%s)', str(settingsDict['REMOVE_ORPHANS']), 'REMOVE_ORPHANS')  
-    logger.info('%s | Removing slow downloads (%s)', str(settingsDict['REMOVE_SLOW']), 'REMOVE_SLOW')
-    logger.info('%s | Removing stalled downloads (%s)', str(settingsDict['REMOVE_STALLED']), 'REMOVE_STALLED')
-    logger.info('%s | Removing downloads belonging to unmonitored items (%s)', str(settingsDict['REMOVE_UNMONITORED']), 'REMOVE_UNMONITORED') 
-    logger.info('')          
-    logger.info('Running every: %s', fmt.format(rd(minutes=settingsDict['REMOVE_TIMER'])))  
-    if settingsDict['REMOVE_SLOW']: 
-        logger.info('Minimum speed enforced: %s KB/s', str(settingsDict['MIN_DOWNLOAD_SPEED'])) 
-    logger.info('Permitted number of times before stalled/missing metadata/slow downloads are removed: %s', str(settingsDict['PERMITTED_ATTEMPTS']))      
-    if settingsDict['QBITTORRENT_URL']: 
-        logger.info('Downloads with this tag will be skipped: \"%s\"', settingsDict['NO_STALLED_REMOVAL_QBIT_TAG'])  
-        logger.info('Private Trackers will be skipped: %s', settingsDict['IGNORE_PRIVATE_TRACKERS'])        
-    
-    logger.info('') 
-    logger.info('*** Configured Instances ***')
-    
-    for instance in settingsDict['INSTANCES']:
-        if settingsDict[instance + '_URL']: 
-            logger.info('%s: %s', settingsDict[instance + '_NAME'], settingsDict[instance + '_URL'])   
-
-    if settingsDict['QBITTORRENT_URL']: logger.info('qBittorrent: %s', settingsDict['QBITTORRENT_URL'])    
-
-    logger.info('') 
-    return   
+    logger.info(
```

Comment:
As written below - pls revert the format changes. I know it doesn't uphold currently to certain coding standards (ie too long lines etc), but I feel it was easier to read before.

Reply:
Will try. Hopefully it's easy to unformat.

Reply:
What I would do is:
- Copy your branch
- Revert your dev branch with git rebase -i XXX
- Delete the commits where the cleanup took place
- git push force
- then merge your copied branch into dev branch (either manually by copying over the relevant code, or litterally a merge where you don't merge the format changes)


maybe there's a more clever way..